test1-dev.js

```diff
@@ -3,3 +3,10 @@ var foo = '' ;
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+var fooOff = '' ;
+
+if(fooOff  === "bar") {
+  console.log( 'hery' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-test-dev/pullRequest?prid=1985)
```suggestion
  console.log( "hery" )
```